content/upgrade_server.md

```diff
@@ -14,60 +14,70 @@ aliases = ["/upgrade_server.html"]
 
 [\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/upgrade_server.md)
 
+Chef Infra Server 14 uses Elasticsearch as its search index.
```

Comment:
should the information about EULA License Agreement and user mot updating from 13 would need to accept the Licence be mentioned here? Since we permit upgrades from 13.17.15 to 14.x

Reply:
OK